smithy-model/src/test/resources/software/amazon/smithy/model/loader/invalid/elided-type-conflicts.smithy

```diff
@@ -0,0 +1,19 @@
+// smithy.example#Invalid$id: Member conflicts with an inherited mixin member: smithy.example#MixinStructure$id
```

Comment:
This is a good test case! But to be clear -- this validation is knows nothing about elision right? It's like an orthogonal validation step for ensuring mixins are sound?

Reply:
Yes. By the time we get to this validator, the member looks just like a normal member, whether the target was set based on a mixin or a resource.